plots/_plotfuncs.py

```diff
@@ -475,7 +667,7 @@ def events(observer, lsd):
 
         tt = observer.transit_times(body, st, et)
 
-        if tt:
+        if tt.size > 0:
             tt = tt[0]
         else:
             continue
```

Comment:
The next line (line 675) needs to change from:
```
      sf_time = ephemeris.unix_to_skyfield_time(tt)
```
to
```
      sf_time = ctime.unix_to_skyfield_time(tt)
```

And, below that, line 690:
```
        window_sec = window_deg * 240.0 * ephemeris.SIDEREAL_S
```
should be
```
        window_sec = window_deg * 240.0 * ctime.SIDEREAL_S
```